spec/features/snap_application_with_maximum_info_spec.rb

```diff
@@ -213,10 +213,10 @@ def upload_documents
   def add_document_photo(url)
     input = %(<input type="hidden" name="step[documents][]" value="#{url}">)
     page.execute_script(
-      <<~eos
+      <<~JAVASCRIPT
         document.querySelector('[data-documents-form]').
           insertAdjacentHTML('beforeend', '#{input}')
-      eos
+      JAVASCRIPT
```

Comment:
oh interesting... I prefer this, I believe.

Reply:
Yeah now rubocop wants 'meaningful' heredoc delimiters https://github.com/bbatsov/rubocop/pull/4467/files

(so anything other than EOS)